packages/mdc-layout-grid/_mixins.scss

```diff
@@ -122,3 +122,17 @@
     align-self: stretch;
   }
 }
+
+@mixin mdc-layout-grid-fixed-column-width($size, $margin, $gutter, $column-width) {
```

Comment:
I feel the original code meant `screen-size` and I don't think it is vague. Since we both documented that clearly and do the type check, I feel fine to leave just as it is. WDYT?